docs/WatchPlugins.md

```diff
@@ -221,8 +229,16 @@ Any key not used by built-in functionality can be claimed, as you would expect.
 
 Should your plugin attempt to overwrite a reserved key, Jest will error out with a descriptive message, something like:
 
-> Watch plugin YourFaultyPlugin attempted to register key `q`, that is reserved internally for quitting watch mode. Please change the configuration key for this plugin.
+:::caution
+
+Watch plugin YourFaultyPlugin attempted to register key `q`, that is reserved internally for quitting watch mode. Please change the configuration key for this plugin.
+
+:::
```

Comment:
This must be a [code block](https://docusaurus.io/docs/markdown-features/code-blocks), not an admonition. Tricky to provide a suggestion. I have to add spaces at the beginning of each line to work around, just remove them:

```suggestion
    ```bash

    Watch plugin YourFaultyPlugin attempted to register key `q`, that is reserved internally for quitting watch mode. Please change the configuration key for this plugin.

    ```
```

Reply:
Added this to the most recent commit.